### DISCUSSION TITLE
Auto indenting support?

### DISCUSSION THREAD

**Discussion by yixin0829** [tamlok/vnote#1707] [Q&A] [answered]
hii is it possible to have vnote support auto indent? after some research only found "indent first line".

update: would also be great if vim indent binding `shift + >>` could be added.

**Comment by tamlok** [comment, marked as answer]
Will add it in the future. VNote 2.0 supports the `Shirt+>` indentation. VNote 3.0 will support it later, too.

**Reply by tamlok** [reply]
auto indent is supported now since 3.3.0

**Reply by yixin0829** [reply]
absolutely amazing! lovin it
Thank you and expecting `shift >>` now

**Reply by tamlok** [reply]
Fixed.
